content/chainguard/chainguard-images/getting-started/php.md

```diff
@@ -134,35 +122,18 @@ The command should output a random name combination:
 ```shell
 ludicrous-walrus
 ```
+In the next step, you'll build the application in a multi-stage Dockerfile.
 
-The demo application is now ready. In the next step, you'll create a Dockerfile to run your app.
-
-## Step 2: Creating the Dockerfile
+## 2. Building a Distroless Image for the Application
 
-To make sure our final image is _distroless_ while still being able to install dependencies with Composer, our build will consist of **two** stages: first, we'll build the application using the `dev` image variant, a Wolfi-based image that includes Composer and other useful tools for development.
-Then, we'll create a separate stage for the final image. The resulting image will be based on the distroless PHP Wolfi image, which means it doesn't come with Composer or even a shell.
+We'll now build a distroless image for the application. To be able to install dependencies with Composer, our build will consist of **two** stages: first, we'll build the application using the `dev` image variant, a Wolfi-based image that includes Composer and other useful tools for development. Then, we'll create a separate stage for the final image. The resulting image will be based on the distroless PHP Wolfi image, which means it doesn't come with Composer or even a shell.
 
-Create a Dockerfile with:
-
-```shell
-touch Dockerfile
-```
-
-Then open this file in your code editor of choice, for example `nano`:
+Open the `Dockerfile` included within the root of the application folder. You can use your code editor of choice, for example `nano`:
```

Comment:
I find this a little awkward, because there's no purpose to opening up the Dockerfile since it's included in the example and you show the contents after the command.

I'd delete this sentence and the `nano` command